starters/gatsby-wordpress-starter/__tests__/snapshotTests/__snapshots__/pagesIndex.test.jsx.snap

```diff
@@ -68,7 +68,70 @@ exports[`<PageIndexTemplate /> > should render with pages 1`] = `
           Pages
         </h1>
       </header>
-      <section />
+      <div
+        class="max-w-screen-lg mx-auto"
+      >
+        <section>
+          <div
+            class="max-w-full"
+          >
+            <h3
```

Comment:
I must've missed this on the paginator PR, but we shouldn't use an h3 here as it is not semantic. This should probably be a `span`